src/tui.rs

```diff
@@ -653,6 +654,9 @@ impl Tui {
         // out where is to do a binary search of commenting out code in main.rs.
         debug_assert!(ctx.tree.current_node.borrow().stack_parent.is_none());
 
+        // End the root node.
+        ctx.block_end();
```

Comment:
Ensures that tab handling works inside the root node (the main window, so to speak). I modified the tree handling code to accommodate for that.